### PR TITLE
TSK-1831: removed JDK8 and integrated wildfly-example module properly

### PR DESCRIPTION
**Pull request by mustaphazorgati**
updated docs:
* https://taskana.atlassian.net/wiki/spaces/TAS/pages/1821935692/Continuous+Integration+GitHub+Actions
* https://taskana.atlassian.net/wiki/spaces/TAS/pages/1758593073/Supported+Environments

sonar: 
* https://sonarcloud.io/summary/new_code?id=mustaphazorgati_taskana&branch=task%2F1831

---
 
### For the submitter:
- [x] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [x] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Review by knht — APPROVED** [review]

File: docker-databases/prepare_db.bat [hunk 1]
```diff
@@ -73,13 +63,4 @@ SET PROP_FILE=%HOMEPATH%\taskanaUnitTest.properties
     ECHO docker-compose -f %~dp0/docker-compose.yml down -v
     docker-compose -f %~dp0/docker-compose.yml down -v
     ECHO ---
-    GOTO REMOVE_PROP
```
Comment (ryzheboka) [inline docker-databases/prepare_db.bat:76]: We probably also need to remove (or replace with "GOTO MENU") "GOTO REMOVE_PROP" in lines 59, 43 and 27. We also should remove "ECHO 6 - Remove %PROP_FILE%" in line 18 and "SET PROP_FILE=%HOMEPATH%\taskanaUnitTest.properties" in line 3.

File: rest/taskana-rest-spring-example-wildfly/src/test/java/pro/taskana/example/wildfly/AbstractAccTest.java [hunk 2]
```diff
@@ -78,4 +85,52 @@ protected String parseServerLog() throws Exception {
...
     return stringBuilder.toString();
   }
+
+  private static void stopPostgresDb() {
+    try {
+      boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
+      ProcessBuilder builder = new ProcessBuilder();
+      if (isWindows) {
+        builder.command(
+            "cmd.exe", "/c", "docker-compose -f ../../docker-databases/docker-compose.yml down");
```
Comment (ryzheboka) [inline rest/taskana-rest-spring-example-wildfly/src/test/java/pro/taskana/example/wildfly/AbstractAccTest.java:95]: Do we need the "-v" only for start, or do we need it here too? ("... down -v")

File: docker-databases/prepare_db.sh [hunk 3]
```diff
@@ -56,29 +56,15 @@ function main() {
     ;;
   DB2|DB2_11_5)
     docker-compose -f $scriptDir/docker-compose.yml up -d "$(mapDBToDockerComposeServiceName "$1")"
-
-    echo 'jdbcDriver=com.ibm.db2.jcc.DB2Driver' > $propFile
```
Comment (ryzheboka) [inline docker-databases/prepare_db.sh:60]: Do we need to remove other usages of propFile like `propFile="$HOME/taskanaUnitTest.properties"` in line 50 ?

File: rest/taskana-rest-spring-example-wildfly/src/test/java/pro/taskana/example/wildfly/AbstractAccTest.java (hunk 2 above)
Comment (mustaphazorgati) [reply rest/taskana-rest-spring-example-wildfly/src/test/java/pro/taskana/example/wildfly/AbstractAccTest.java]: good catch. Was a mistake. We only need `down -v`.

File: docker-databases/prepare_db.bat (hunk 1 above)
Comment (mustaphazorgati) [reply docker-databases/prepare_db.bat:76]: good catch. Done.

File: docker-databases/prepare_db.sh (hunk 3 above)
Comment (mustaphazorgati) [reply docker-databases/prepare_db.sh:60]: good catch. Done!

**Review by ryzheboka — DISMISSED** [review]

**Review by mustaphazorgati — COMMENTED** [review]
LGTM

**Review by knht — DISMISSED** [review]
LGTM!

**Review by ryzheboka — APPROVED** [review]

**Pull request merged**